jest.config.js

```diff
@@ -79,7 +79,9 @@ const testVenia = inPackage => ({
     // @magento namespaced packages like Peregrine and Venia UI as well, when
     // it's testing Venia. That way, changes in sibling packages don't require a
     // full compile.
-    transformIgnorePatterns: ['node_modules/(?!@magento/)'],
+    transformIgnorePatterns: [
+        'node_modules/(?!@magento|jarallax|video-worker/)'
```

Comment:
Yup, we include a library called Jarallax (ironically for Parallax). We opted to use this library rather than a dedicated React library so we could easily achieve feature parity.

Reply:
Sure. Does Jarallax need to be transpiled by `babel-jest` though? Ordinarily any third-party code in `node_modules` doesn't need to be, so I'm not sure why these two libraries would.

Reply:
`video-worker` is a dep of Jarallax, so they came in through the same feature. 

This is because Jarallax is published _very weirdly_, their index file points out their `src` which has ES6 code within. Without this entry into `transformIgnorePatterns` it errors on the `import` statements within the library.

```
const jarallax = require('./src/jarallax.esm').default;
const jarallaxVideo = require('./src/jarallax-video.esm').default;
const jarallaxElement = require('./src/jarallax-element.esm').default;

module.exports = {
    jarallax,
    jarallaxElement() {
        return jarallaxElement(jarallax);
    },
    jarallaxVideo() {
        return jarallaxVideo(jarallax);
    },
};

```

Reply:
That's the answer I was looking for, thanks. 👍